identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/screens/MainScreen.kt

```diff
@@ -53,13 +74,24 @@ fun MainScreen(sampleName: String, application: Application) {
                     .fillMaxSize()
                     .padding(it)
             ) {
-                // composable function that wraps the MapView
-                ComposeMapView(
+                MapView(
                     modifier = Modifier
                         .fillMaxSize()
                         .weight(1f)
                         .animateContentSize(),
-                    mapViewModel = mapViewModel
+                    arcGISMap = mapViewModel.arcGISMap,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = viewpoint),
+                    mapViewProxy = mapViewProxy,
+                    onSingleTapConfirmed = { singleTapConfirmedEvent ->
+                        sampleCoroutineScope.launch {
+                            val identifyResult = mapViewProxy.identifyLayers(
+                                screenCoordinate = singleTapConfirmedEvent.screenCoordinate,
+                                tolerance = 12.dp,
+                                maximumResults = 10
+                            )
+                            mapViewModel.handleIdentifyResult(identifyResult)
```

Comment:
comment here as well: 
```suggestion
                            // use the layer result to display feature information 
                            mapViewModel.handleIdentifyResult(identifyResult)
```